packages/ragbits-document-search/src/ragbits/document_search/documents/element.py

```diff
@@ -42,15 +43,14 @@ def id(self) -> str:
         id_components = [
             self.document_meta.id,
             self.element_type,
-            self.key,
-            self.text_representation,
+            self.key or "null",
```

Comment:
Do `str(self.key)` to use Python's built-in `None` keyword instead of a magic string that isn't defined anyway and as such is quite brittle. If you really want to use the string "null", declare a constant - but personally I don't think that's necessary.

Reply:
done